static/templates/search_list_item.hbs

```diff
@@ -0,0 +1,8 @@
+<div class="search_list_item">
+    {{{ description }}}
```

Comment:
I think this should probably be called `description_html`, for clarity that it is being inserted without further escaping?